daemon/web/components/TerminalView/TerminalView.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './index.sass';
+
+const styles = {
+  textarea: {
+    position: 'relative',
+    top: '1px',
+    width: '100%',
+    height: '70px',
+    color: '#ffffff',
+    backgroundColor: '#212b36',
+    fontSize: '10px',
+    resize: 'none',
+  },
+};
+
+const TerminalViewView = props =>
+  (
+    <div>
+      <textarea
+        style={styles.textarea}
+        readOnly
+        value={props.logs.reduce((accumulator, currentVal) => accumulator + '\r\n' + currentVal)} />
+    </div>
+  );
+
+
+TerminalViewView.propTypes = {
+  logs: PropTypes.string,
```

Comment:
in `App.js`, `const mocklogs` is an array, but you pass it in as prop `logs`:

```js
        <TerminalView logs={mocklogs} />
```

This assertion should assert that the prop is an array, and not a string

Reply:
oops forgot to change this one